__tests__/auth.js

```diff
@@ -31,7 +34,22 @@ describe('Authentication', () => {
     response.resetMocked();
   });
 
-  beforeAll(() => osjs().then(c => (core = c)));
+  beforeAll(() =>
+    osjs().then(async c => {
+      core = c;
+      c.make('osjs/fs');
+      filesystem = new Filesystem(core);
+      filesystem.init();
```

Comment:
The `init()` method is async. Add `await` to this in order to avoid any weird side-effects.